examples/assured-workloads-example/main.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Google LLC
```

Comment:
Need an example readme

Reply:
I'm using the same structure of other examples in this repositories: they have the readme in the "root" folder, not in inside the "examples" folder.

Anyways, do you think we should move and adapt the README from `kms-solutions/assured-workloads/` to `kms-solutions/examples/assured-workloads-example/`? In case we decide to have readme's in each of these folders, let's discuss the content of each one in our next sync?

Reply:
As we have talked, the example and the "module" now have a README.